index.html

```diff
@@ -2113,22 +2112,23 @@
 for a set of [=purposes=] (that they typically share with other [=people=] who interact with the
 same environment).
 
-## Server-Side Parties {#parties}
+## Server-Side Actors {#parties}
```

Comment:
This is a nitpick, but since this document hasn't yet been referred to much I think we could decide to break that link.